### PR TITLE
Vladimir Shevtsov 11 node frameworks

### PR DESCRIPTION
**Pull request by shevv920**

### CODE REVIEW

File: 11-node-frameworks/classworks/Shevtsov_Vladimir/src/koa-app.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+const xtpl = require('koa-xtpl');
+const bodyParser = require('koa-bodyparser');
+const Joi = require('joi');
+
+app.keys = ['keys', 'asd'];
+app.use(session());
+
+
+const index = async (ctx, next) => {
+  await ctx.render('index', { sessionCounter: ctx.state.sessionCounter });
```
Comment (anstill) [inline 11-node-frameworks/classworks/Shevtsov_Vladimir/src/koa-app.js:17]: 👍 for using server-side rendering and templates

File: 11-node-frameworks/classworks/Shevtsov_Vladimir/src/koa-app.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+router.get('/', index);
+router.get('/form', form);
+
+const schema = Joi.object({ lastName: Joi.string().required(), descr: Joi.string().min(3).required() });
+
+router.post(
+  '/summer',
+  schema.validate(ctx.body),
```
Comment (anstill) [inline 11-node-frameworks/classworks/Shevtsov_Vladimir/src/koa-app.js]: I'm not sure where do you get `ctx.body`. Seems like that will occur an error

**Review by anstill — CHANGES_REQUESTED** [review]
hey @shevv920, great job here! Check my comments and feel free to ask any question. 

Would b great if you add the possibility to get data from sent user forms.

File: 11-node-frameworks/classworks/Shevtsov_Vladimir/src/koa-app.js (hunk 2 above)
Comment (shevv920) [reply 11-node-frameworks/classworks/Shevtsov_Vladimir/src/koa-app.js]: fixed

**Review by anstill — APPROVED** [review]
🚀

**Pull request merged**